operator-pipeline-images/operatorcert/entrypoints/validate_catalog_format.py

```diff
@@ -0,0 +1,120 @@
+"""Validate catalog format for all given catalogs."""
+
+import argparse
+import logging
+from subprocess import CalledProcessError
+import sys
+
+from operatorcert.logger import setup_logger
+from operatorcert.utils import SplitArgs, run_command
+
+LOGGER = logging.getLogger("operator-cert")
+
+
+def setup_argparser() -> argparse.ArgumentParser:
+    """
+    Setup argument parser
+
+    Returns:
+        argparse.ArgumentParser: Argument parser
+    """
+    parser = argparse.ArgumentParser(
+        description="Validate catalog format for all given catalogs."
+    )
+    parser.add_argument(
+        "--repo-path",
+        default=".",
+        help="Path to the root of the local clone of the repo",
+        required=True,
+    )
+    parser.add_argument(
+        "--catalog-names",
+        default=[],
+        action=SplitArgs,
+        help="List of catalog names to validate",
+    )
+    parser.add_argument(
+        "--makefile-path",
+        help="Path to the operator Makefile",
+        required=True,
+    )
+
+    parser.add_argument("--verbose", action="store_true", help="Verbose output")
+
+    return parser
+
+
+def run_makefile(makefile_path: str) -> None:
+    """Run makefile"""
+    cmd = [
+        "make",
```

Comment:
The makefile target should be `make catalog` based on a template given to us by opm team.

Reply:
Changed, thanks for pointing out.